qt/widgets/common/src/MuonFitPropertyBrowser.cpp

```diff
@@ -1063,6 +887,66 @@ bool MuonFitPropertyBrowser::isWorkspaceValid(Workspace_sptr ws) const {
 }
 
 void MuonFitPropertyBrowser::finishHandle(const IAlgorithm *alg) {
+  if (alg->name() == "CalculateMuonAsymmetry") {
+    finishHandleTF(alg);
+  } else {
+    finishHandleNormal(alg);
+  }
+}
+void MuonFitPropertyBrowser::finishHandleTF(const IAlgorithm *alg) {
+
```

Comment:
There is a lot of common code between `finishHandleNormal` and `finishHandleTF` that can be turned into function calls.